cvat-ui/src/components/cvat-app.tsx

```diff
@@ -456,7 +471,40 @@ class CVATApplication extends React.PureComponent<CVATAppProps & RouteComponentP
                 }
             }
         }
-
+        const bulkError = notifications.errors.selection.bulkOperation;
```

Comment:
I believe bulk-progress component should be responsible for this logic. Like:

... progress ...
... N items failed, repeat? ... 

What do you think?